models/op/token_distributions/optimism/op_token_distributions_optimism_project_wallets.sql

```diff
@@ -15,61 +16,66 @@
 with intermediate_wallets AS (
 SELECT address, proposal_name, address_descriptor
 FROM (
-SELECT LOWER(address) AS address, cast(proposal_name as varchar(100)) AS proposal_name, cast(address_descriptor as varchar(100)) AS address_descriptor
+SELECT address, cast(proposal_name as varchar) AS proposal_name, cast(address_descriptor as varchar) AS address_descriptor
```

Comment:
likely don't need cast

Reply:
keeping in case we use '0x' as a project name